app/src/test/java/com/github/epfl/meili/helpers/DateAuxiliaryTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.github.epfl.meili.helpers
```

Comment:
This is a little bit weird, it seems like the changes from previous PRs are being committed again. Do you have any idea of what might be going wrong and how to fix it?

Reply:
I can't rebase anymore, there are way too many commits. I tried for 2 hours to find a solution to it and mine was to just copy the changes from main to make sure there aren't going to be any conflicts.

Reply:
Like if I try to rebase, there are conflicts on every single commit, I don't know why

Reply:
I have way too many commits and when I rebase there are conflicts on every single one of them. I tried squashing and other git stuff without success. my solution was to just make sure there aren't going to be any conflicts when merging.